test_suite/tests/derive.rs

```diff
@@ -867,3 +867,24 @@ fn ranges() {
 
     assert_type!(Rangey, ty);
 }
+
+#[test]
+fn strips_invisible_delimiters_from_type_name() {
+    // When a type is specified as part of a macro, the tokens passed to the `TypeInfo` derive macro
+    // are a type `Group`, which is pretty printed with invisible delimiters e.g. /*«*/ bool /*»*/.
```

Comment:
Maybe move/copy this comment into the code as well? Seems like a non-obvious gotcha?